src/components/ActivityFeed/ActivityFeedCalendar.tsx

```
@@ -0,0 +1,179 @@
+		likes?: number;
+		dislikes?: number;
+	}
+
+	const [activityData, setActivityData] = useState<ActivityData[]>([]);
+	const [loading, setLoading] = useState(true);
+
+	const fetchActivityCalendar = async () => {
+		try {
+			setLoading(true);
+			const { data } = await nextApiClientFetch<any>('/api/v1/activity-feed/activity-calendar');
+			if (data && data.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)